cdap-docs/developers-manual/source/advanced/application-logback.rst

```diff
@@ -0,0 +1,43 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _application-logback:
+
+====================
+Application Logbacks
+====================
+
+YARN containers spawned by a CDAP application use the default container logback, ``logback-container.xml``,
```

Comment:
We could, but in lieu, let's add a glossary entry to ``cdap-docs/reference-manual/source/glossary.rst``.
Please edit/correct this as appropriate:

    Master Services
         The YARN container with the system services started by CDAP.